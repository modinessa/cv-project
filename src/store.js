import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
