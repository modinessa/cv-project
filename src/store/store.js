import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});