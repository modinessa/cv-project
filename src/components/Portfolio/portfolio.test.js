import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { Portfolio } from './index';

const TEST_FILTERS = [{category: 'All', isChecked: true}];
const TEST_CARDS = [{
    id: 1,
		img: 'someImg.png',
    category:'code',
		portfolioInfo: {
			title: 'Some text',
			text: 'Donec pede justo, fringilla vel, aliquet nec, vulel, aliquet nec, vulputate eget eget, arcu. In enim justo',
			url: 'https://somesite.com'
		}
  }];


describe('Portfolio', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<Portfolio />
			</Provider>
		)).toMatchSnapshot();
	})
})