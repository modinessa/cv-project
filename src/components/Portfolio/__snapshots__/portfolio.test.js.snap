// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio renders component 1`] = `
<section
  className="app-section"
>
  <h2>
    Portfolio
  </h2>
  <div
    className="filter-container"
  >
    <div
      className="filter-item"
    >
      <div
        className="filter isChecked"
        id="All"
        onClick={[Function]}
      >
        All
      </div>
      <div
        className="separator"
      >
        /
      </div>
    </div>
    <div
      className="filter-item"
    >
      <div
        className="filter false"
        id="Code"
        onClick={[Function]}
      >
        Code
      </div>
      <div
        className="separator"
      >
        /
      </div>
    </div>
    <div
      className="filter-item"
    >
      <div
        className="filter false"
        id="UI"
        onClick={[Function]}
      >
        UI
      </div>
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="card code"
      style={
        Object {
          "background": "url(code.png)",
        }
      }
    >
      <div
        className="card-info-container"
      >
        <div
          className="card-info"
        >
          <h4>
            Some text
          </h4>
          <p>
            Donec pede justo, fringilla vel, aliquet nec, vulel, aliquet nec, vulputate eget eget, arcu. In enim justo
          </p>
          <a
            href="https://somesite.com"
          >
            View resource
          </a>
        </div>
      </div>
    </div>
    <div
      className="card ui"
      style={
        Object {
          "background": "url(ui.png)",
        }
      }
    >
      <div
        className="card-info-container"
      >
        <div
          className="card-info"
        >
          <h4>
            Title
          </h4>
          <p>
            Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo
          </p>
          <a
            href="https://somesite.com"
          >
            View resource
          </a>
        </div>
      </div>
    </div>
    <div
      className="card code"
      style={
        Object {
          "background": "url(code.png)",
        }
      }
    >
      <div
        className="card-info-container"
      >
        <div
          className="card-info"
        >
          <h4>
            Bla-bla
          </h4>
          <p>
            Pede justo, fringilla vel, aliquet nec, vulputate eget,vel, aliquet nec, vulputate eget arcu. In enim justo
          </p>
          <a
            href="https://somesite.com"
          >
            View resource
          </a>
        </div>
      </div>
    </div>
    <div
      className="card ui"
      style={
        Object {
          "background": "url(ui.png)",
        }
      }
    >
      <div
        className="card-info-container"
      >
        <div
          className="card-info"
        >
          <h4>
            Awesome text
          </h4>
          <p>
            Donec pede jusaliquet necto, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo
          </p>
          <a
            href="https://somesite.com"
          >
            View resource
          </a>
        </div>
      </div>
    </div>
  </div>
</section>
`;
